bin/mysql-dump-anonymize.php

```diff
@@ -7,6 +7,6 @@
 require_once dirname(__DIR__).'/vendor/autoload.php';
 
 error_reporting(E_ALL);
-ini_set('display_errors', '1');
+ini_set('display_errors', 'stderr');
```

Comment:
removed the @ from @userialize so notices are still outputed, but not in the dump

Reply:
Actually, I don't agree with that.
As we handle the result of the `unserialize` programmatically, there is no need to generate a notice as it's of no use for debugging and I think there is no action to act upon further.

What I think we can do more is to improve the result check:

Reply:
We discussed offline and actually the fallback is not considered a feature but just a failure fallback and user should be aware about it and either fix the dumping in case that caused issues or change the column type if if was chosen by error as serialized data.